netflix-roulette/src/pages/home/components/Catalog/Catalog.styles.ts

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
```

Comment:
Might be worth storing your re-usable styled components in some common place, to avoid something, like `import { CatalogContainer } from '../../../../../../../ ...`

Reply:
CatalogContainer used only once, all reusable components are stored in UI folder